src/crypto/crypto_x509.cc

```diff
@@ -215,24 +387,10 @@ void KeyUsage(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
   X509Certificate* cert;
   ASSIGN_OR_RETURN_UNWRAP(&cert, args.This());
-
-  auto eku = cert->view().getKeyUsage();
-  if (!eku) return;
-
-  const int count = sk_ASN1_OBJECT_num(eku.get());
-  MaybeStackBuffer<Local<Value>, 16> ext_key_usage(count);
-  char buf[256];
-
-  int j = 0;
-  for (int i = 0; i < count; i++) {
-    if (OBJ_obj2txt(buf, sizeof(buf), sk_ASN1_OBJECT_value(eku.get(), i), 1) >=
-        0) {
-      ext_key_usage[j++] = OneByteString(env->isolate(), buf);
-    }
+  Local<Value> ret;
+  if (GetKeyUsage(env, cert->view()).ToLocal(&ret)) {
+    args.GetReturnValue().Set(ret);
```

Comment:
Nit: Maybe?

```suggestion
  Local<Value> ret = Undefined(env->isolate());
  USE(GetKeyUsage(env, cert->view()).ToLocal(&ret));
  args.GetReturnValue().Set(ret);
```

Reply:
This is unnecessary. If `ToLocal(...)` fails here it is likely because of an error. We want to allow that to propagate through so we don't actually want to set the return value if that's the case.